BlueCombing/Views/MyActivityView/MyActivityView.swift

```diff
@@ -124,14 +135,14 @@ struct MyActivityView: View {
 }
 
 
-struct representBadgeView : View{
-    var representBadge: String
-    @State var description: String = "산호초"
-    @State var BadgeTitle: String = "img_badge_coral"
+struct presentBadgeView : View{
+    var presentBadge: String
+    @State var description: String = ""
+    @State var badgeImage: UIImage = UIImage(named: "img_badge_coral")!
```

Comment:
```suggestion
    @State var badgeImage: = UIImage(named: "img_badge_coral")!
```
이 부분은 충분하게 선언으로 자료형을 알 수 있음으로 자료형을 명시할 필요가 없을 것 같습니다!

Reply:
말씀해 주신 부분 수정 했습니다. 하나 여쭤보고 싶은게 있는데, 혹시 자료형을 명시하고 명시하지 않고 에 대한 기준이 따로 있을까요??

Reply:
Swift는 설계에 의한 안정성에 의해서 타입이 명확하다면 명시하지 않도록 설계된 언어로 알고 있습니다.

또한 코드 가독성 측면에서도 자료형을 누가봐도 알 수 있다면 안 적는 것이 좋다고 알고 있습니다.

Reply:
그렇군요! 감사합니다!